repair/repair.cc

```diff
@@ -1478,3 +1491,52 @@ future<> repair_abort_all(seastar::sharded<database>& db) {
         repair_tracker().abort_all_repairs();
     });
 }
+
+future<> sync_data_with_repair(seastar::sharded<database>& db,
```

Comment:
Suggest s/with/using/. "with" implies we're synchronizing some data with the repair process.

Reply:
Done.